### PR TITLE
gnome-nibbles: fix compile

### LINKED ISSUE #5555: [gnome_nibbles] Failed to build
```
FAILED: src/gnome-nibbles.p/boni.c src/gnome-nibbles.p/controls.c src/gnome-nibbles.p/gnome-nibbles.c src/gnome-nibbles.p/nibbles-game.c src/gnome-nibbles.p/nibbles-view.c src/gnome-nibbles.p/nibbles-window.c src/gnome-nibbles.p/players.c src/gnome-nibbles.p/preferences-dialog.c src/gnome-nibbles.p/scoreboard.c src/gnome-nibbles.p/speed.c src/gnome-nibbles.p/warp.c src/gnome-nibbles.p/worm.c 
valac -C --pkg posix --pkg libgnome-games-support-1 --pkg gtk+-3.0 --pkg gsound --target-glib ' 2.42.0' --pkg glib-2.0 --pkg gio-2.0 --pkg gee-0.8 --pkg clutter-gtk-1.0 --pkg clutter-1.0 --color=always --directory src/gnome-nibbles.p --basedir ../src --gresources=../data/gnome-nibbles.gresource.xml --vapidir /usr/local/tmp/crew/gnome-nibbles-3.38.2.tar.xz.dir/gnome-nibbles-3.38.2/src/vapi ../src/vapi/config.vapi ../src/boni.vala ../src/controls.vala ../src/gnome-nibbles.vala ../src/nibbles-game.vala ../src/nibbles-view.vala ../src/nibbles-window.vala ../src/players.vala ../src/preferences-dialog.vala ../src/scoreboard.vala ../src/speed.vala ../src/warp.vala ../src/worm.vala
error: Package `gsound' not found in specified Vala API directories or GObject-Introspection GIR directories
Compilation failed: 1 error(s), 0 warning(s)
[6/26] Compiling Vala source ../src/boni.vala ../src/nibbles-game.vala ../src/nibbles-test.vala ../src/worm.vala ../src/warp.vala
../src/nibbles-game.vala:287.5-287.23: warning: Method `NibblesGame.pause' never used
    internal void pause ()
    ^^^^^^^^^^^^^^^^^^^
../src/nibbles-game.vala:293.5-293.25: warning: Method `NibblesGame.unpause' never used
    internal void unpause ()
    ^^^^^^^^^^^^^^^^^^^^^
../src/nibbles-game.vala:299.5-299.30: warning: Method `NibblesGame.reset' never used
    internal inline void reset (int start_level)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^
../src/nibbles-game.vala:698.5-698.38: warning: Method `NibblesGame.load_worm_properties' never used
    internal void load_worm_properties (Gee.ArrayList<Settings> worm_settings)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
../src/nibbles-game.vala:746.5-746.33: warning: Method `NibblesGame.handle_keypress' never used
    internal bool handle_keypress (uint keyval)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Compilation succeeded - 5 warning(s)
ninja: build stopped: subcommand failed.
gnome_nibbles failed to build: `ninja -C builddir` exited with 1
```

### PR DESCRIPTION
**Pull request by supechicken**
Switched source to git

Fixes #5555

### CODE REVIEW

File: packages/gnome_nibbles.rb [hunk 1]
```diff
@@ -1,45 +1,34 @@
 require 'package'
 
 class Gnome_nibbles < Package
+  @_commit = '62964e9256fcac616109af874dbb2bd8342a9853'
```
Comment (uberhacker) [inline packages/gnome_nibbles.rb:4]: Can we move this line closer to the `source_url` line?  I'd like to reserve the top for description, homepage, version, license and compatibility.  This also maintains consistency in packages which can help avoid confusion.

File: packages/gnome_nibbles.rb [hunk 2]
```diff
@@ -1,45 +1,34 @@
...
-    FileUtils.mv "#{CREW_DEST_PREFIX}/bin/#{@_app}", "#{CREW_DEST_PREFIX}/bin/#{@_app}.elf"
-    FileUtils.install @_app, "#{CREW_DEST_PREFIX}/bin/#{@_app}", mode: 0755
   end
 
   def self.postinstall
-    system 'gdk-pixbuf-query-loaders --update-cache'
-    system "glib-compile-schemas #{CREW_PREFIX}/share/glib-2.0/schemas"
+    system 'update-mime-database', "#{CREW_PREFIX}/share/mime"
+    system 'gdk-pixbuf-query-loaders', '--update-cache'
+    system 'glib-compile-schemas', "#{CREW_PREFIX}/share/glib-2.0/schemas"
```
Comment (uberhacker) [inline packages/gnome_nibbles.rb:32]: I assume `gdk_pixbuf`, `glib` and `shared_mime_info` are inherited dependencies?

File: packages/gnome_nibbles.rb (hunk 1 above)
Comment (supechicken) [reply packages/gnome_nibbles.rb]: Done

File: packages/gnome_nibbles.rb (hunk 2 above)
Comment (supechicken) [reply packages/gnome_nibbles.rb:32]: They are the dependencies of gtk3, which is a dependency of libgnome-game-support

**Pull request merged**